cypress/utils/utils.ts

```diff
@@ -1331,3 +1331,7 @@ export function validateTooLongInput(selector, anotherSelector?: string): void {
     if (anotherSelector) click(anotherSelector);
     doesExistText("This field must contain fewer than 120 characters.", true);
 }
+
+export function enumKeys<O extends object, K extends keyof O = keyof O>(obj: O): K[] {
```

Comment:
please put comment inside the method , what it is doing and it's use

Reply:
Comments added